pkg/operator/keys/create.go

```diff
@@ -185,20 +185,95 @@ func saveEcdsaKey(keyName string, p utils.Prompter, privateKey *ecdsa.PrivateKey
 	}
 
 	privateKeyHex := hex.EncodeToString(privateKey.D.Bytes())
-	// TODO: display it using `less` of `vi` so that it is not saved in terminal history
-	fmt.Println("ECDSA Private Key (Hex): ", privateKeyHex)
-	fmt.Println("\033[1;32m🔐 Please backup the above private key hex in a safe place 🔒\033[0m")
-	fmt.Println("Key location: " + fileLoc)
 	publicKey := privateKey.Public()
 	publicKeyECDSA, ok := publicKey.(*ecdsa.PublicKey)
 	if !ok {
-		return err
+		return errors.New("error casting public key to ECDSA public key")
 	}
 	publicKeyBytes := crypto.FromECDSAPub(publicKeyECDSA)
-	fmt.Println("Public Key hex: ", hexutil.Encode(publicKeyBytes)[4:])
+	publicKeyHex := hexutil.Encode(publicKeyBytes)[4:]
 	address := crypto.PubkeyToAddress(*publicKeyECDSA).Hex()
-	fmt.Println("Ethereum Address", address)
-	return nil
+
+	return displayWithLess(fileLoc, privateKeyHex, fileLoc, publicKeyHex, address, KeyTypeECDSA)
+}
+
+func displayWithLess(filePath, privateKeyHex, fileLoc, publicKeyHex, address, keyType string) error {
+    var message, border, keyLine string
+    tabSpace := "    " 
+
+    keyContent := tabSpace + privateKeyHex + tabSpace
+    borderLength := len(keyContent) + 4
+    border = strings.Repeat("/", borderLength)
+    paddingLine := "//" + strings.Repeat(" ", borderLength-4) + "//"
+
+    keyLine = fmt.Sprintf("//%s//", keyContent)
+
+    if keyType == KeyTypeECDSA {
+        message = fmt.Sprintf(`
+ECDSA Private Key (Hex):
```

Comment:
Actually one thing we can improve here is - only private key should be shown in `less` command. The key location and public key could be in the terminal history since that doesn't leak any sensitive info and that information is needed for further actions. wdyt?

Reply:
Yeah no problem! From my perspective, I just initially did it to display everything using the `less` command. However, I can now modify it to ensure that only the private key is displayed. This was my first reaction when I saw the TODO, but did not know what you thought 👍 

Would you then like the key location and public key should be displayed before or after when the private key has been shown?

Reply:
I think after we show the private key we can show the other things in terminal.

Reply:
No problem. Will fix this straight away.